Сверстать экран профиля кодом. Спринт 9

WayOfTheDev opened the pull request.

WayOfTheDev committed:
Сверстать экран профиля кодом. Спринт 9

DanilaG commented on Image Feed/ImagesList/ImagesListViewController.swift:
```diff
@@ -56,13 +58,35 @@ extension ImagesListViewController: UITableViewDataSource {
+        let imageInsets = UIEdgeInsets(top: 4, left: 16, bottom: 4, right: 16)
+        let imageViewWidth = tableView.bounds.width - imageInsets.left - imageInsets.right
+        let imageWidth = image.size.width
+        let scale = imageViewWidth / imageWidth
```
Мало вероятно, но в теории, `imageWidth` может оказаться равным 0 и тогда получим падение приложения при попытки поделить на 0. Давай добавим проверку деления на 0

DanilaG commented on Image Feed/Profile/ProfileViewController.swift:
```diff
@@ -0,0 +1,137 @@
+//  Created by D. K. on 11.09.24.
+//
+
+//import UIKit
```
Давай уберём не нужный код

WayOfTheDev committed:
Исправления:
В проекте кнопка выхада цвета red, в макетах цвета ypRed. Давай сделаем как в макетах - Готово
В макетах кнопка поделиться размером 51 x 51, в проекте 50 x 50. Давай сделаем как в макетах - Готово, сделал 51 на 51, но в макете у меня 50 на 50
